tests/python/tvmscript/test_builder_basic.py

```diff
@@ -173,9 +173,74 @@ def test_builder_for():
     print(b.get().script())
 
 
+def test_builder_stmt():
+    print("test_builder_stmt")
+    with Builder() as b:
+        with T.prim_func():
+            thread_x = def_("thread_x", T.env_thread("threadIdx.x"))
+            thread_y = def_("thread_y", T.env_thread("threadIdx.y"))
+            buffer_x = def_("buffer_x", tvm.tir.decl_buffer([128, 128]))
+            buffer_y = def_("buffer_y", tvm.tir.decl_buffer([128, 128]))
+            var_x = def_("var_x", tvm.tir.Var("", tvm.ir.PrimType("int32")))
+            var_y = def_("var_y", tvm.tir.Var("", tvm.ir.PrimType("int32")))
```

Comment:
As for user interface, let's keep it simple so that users could comprehend.

```suggestion
            var_x = def_("var_x", tvm.tir.Var("", dtype="int32"))
            var_y = def_("var_y", tvm.tir.Var("", dtype="int32")
```